src/main/scala/unicredit/spark/hbase/HFileSupport.scala

```diff
@@ -139,6 +140,81 @@ sealed abstract class HFileRDD extends Serializable {
       fs.deleteOnExit(new Path(TotalOrderPartitioner.getPartitionFile(job.getConfiguration)))
     }
   }
+  /*
+   * Overloaded function, which requires that sequence of headers and sequence of columns are in the sorted 
+   * order as required by HBase (i.e. sorted by columns names) . This avoids creation of TreeSet objects to sort the
+   * columns for every row.
+   * It also avoids creation of the Map objects for each row where there is mapping of a column name to
+   * column value for each row.
+   * If you come across GC overhead issues, try using this method instead of the other overloaded method. 
+   * 
```

Comment:
from doc, it's still not clear what `rdd: RDD[(String, Seq[A])` is, only after digging into the code, I found it's supposed to be  `(key: String, columns: Seq[A])`